source/draw/gpu/opengl/backend/egl/EglDisplayContext.ooc

```diff
@@ -35,21 +35,11 @@ EglDisplayContext: class extends DisplayContext {
 		This validate(eglBindAPI(EGL_OPENGL_ES_API), EGL_TRUE, "eglBindAPI")
 		configAttribs := [
 			EGL_SURFACE_TYPE, EGL_PBUFFER_BIT,
-			EGL_RENDERABLE_TYPE, EGL_OPENGL_ES2_BIT,
-			EGL_BLUE_SIZE, 8,
-			EGL_GREEN_SIZE, 8,
-			EGL_RED_SIZE, 8,
-			EGL_ALPHA_SIZE, 8,
-			EGL_SAMPLES, 0,
-			EGL_DEPTH_SIZE, 0,
-			EGL_BIND_TO_TEXTURE_RGBA, EGL_DONT_CARE,
```

Comment:
You can try if readding these still works on the laptop in order to minimize the risk of breaking other computers.